specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json

```diff
@@ -23507,19 +23811,150 @@
           "type": "object",
           "additionalProperties": {}
         }
-      }
+      },
+      "additionalProperties": {}
     },
     "Source": {
       "description": "Source of satellite data.",
       "enum": [
-        "Sentinel_2_L2A"
+        "Sentinel_2_L2A",
+        "Sentinel_2_L1C"
       ],
       "type": "string",
       "x-ms-enum": {
         "name": "Source",
         "modelAsString": true
       }
     },
+    "StacFeature": {
+      "description": "Schema of STAC Feature(Item).\r\nRefer for spec: https://github.com/radiantearth/stac-spec/blob/v1.0.0-beta.2/item-spec/item-spec.md#item-fields.",
+      "required": [
+        "assets",
+        "id",
+        "links",
+        "properties",
+        "stacVersion",
+        "type"
+      ],
+      "type": "object",
+      "properties": {
+        "stacVersion": {
+          "description": "The STAC version the Feature implements.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "stacExtensions": {
+          "description": "A list of extensions the Feature implements.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "id": {
+          "description": "Provider identifier. Globally unique ID by Data provider.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the GeoJSON Object. It's value is always Feature.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "geometry": {
+          "description": "Defines the full footprint of the asset represented by this item.\r\nIts a GeoJSON geometry."
+        },
+        "bbox": {
+          "description": "Bounding box of the feature.",
+          "type": "array",
+          "items": {
+            "format": "double",
+            "type": "number"
+          }
+        },
+        "properties": {
+          "description": "A dictionary of additional metadata for the item."
+        },
+        "links": {
+          "description": "List of link objects to resources and related URLs.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StacFeatureLink"
+          }
+        },
+        "assets": {
+          "description": "Dictionary of asset objects, each with a unique key.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/StacFeatureAsset"
+          }
+        },
+        "collection": {
```

Comment:
collectionId ?

Reply:
This model's properties are according to an external specification, and it refers to this as "collection" only.